devicon.json

```
@@ -3894,6 +3894,25 @@
         },
         "color": "#dd4814",
         "aliases": []
+    },
+	{
+        "name": "unity",
+        "tags": [
+			"csharp",
+            "engine",
+            "game"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original"
```

I think the `wordmark` version will look great as an icon too.
```suggestion
                "original",
                "original-wordmark"
```